packages/office-ui-fabric-react/src/components/pickers/TagPicker/TagItem.tsx

```diff
@@ -48,7 +48,7 @@ export const TagItemBase = (props: ITagItemProps) => {
         disabled={disabled}
         iconProps={{ iconName: 'Cancel', styles: { root: { fontSize: '12px' } } }}
         className={classNames.close}
-        ariaLabel={removeButtonAriaLabel}
+        ariaLabel={removeButtonAriaLabel || 'Remove'}
```

Comment:
ITT this is only an issue in the demo, I would add the remove label to the demo page instead of adding it here.

Reply:
I assumed that the cancel button should always have an aria label of `Remove` unless specified, since its role is always going to be to remove the item from the tag picker (doesn't really need to be specified on a case-by-case basis). Any thoughts?

Reply:
@suryasu Fabric, for the most part, shouldn't have any strings in it. I don't believe that we provide default strings for any components labels or text.

Reply:
i see. I'll add just add it to the demo page then!